climada/hazard/base.py

```diff
@@ -725,8 +729,10 @@ def strict_positive_int_accessor(array: xr.DataArray) -> np.ndarray:
                 default_value=[
                     to_csr_matrix(
                         xr.apply_ufunc(
-                            lambda x: np.where(x != 0, 1, 0), data[intensity]
-                        ).values
+                            lambda x: np.where(x != 0, 1, 0), data[intensity],
+                            dask="parallelized",
```

Comment:
so the default is multiprocessing - right?
silly question: does this work in a Jupyter Notebook? multiprocessing used to be cumbersome on Jupyter...
(and if so - is it _really_ multiprocessing in there?)

Reply:
Woops, this is left over from before the new fraction handling. With the new constructor (#541), we do not need this anymore and I actually already adapted this in #589. So I'll merge `develop` into this branch again and resolve this.

About your question: Without explicitly constructing a `dask.distributed` cluster in the background, dask tries to parallelize using multithreading. So the parallelization here would come at no cost but with the benefit of potentially exploiting multithreading capabilities of the machine